lib/rubocop/cop/mixin/active_record_helper.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # A mixin to extend cops for ActiveRecord features
```

Comment:
The proper notation is `Active Record` in this case. Can you update it in this PR?

> The proper names of Rails components have a space in between the words, like "Active Support". ActiveRecord is a Ruby module, whereas Active Record is an ORM.

https://github.com/rails/rails/blob/v6.0.2.1/guides/source/api_documentation_guidelines.md#wording